client/src/rpc_client.rs

```diff
@@ -196,6 +196,34 @@ impl RpcClient {
         })
     }
 
+    pub fn get_block_time(&self, slot: Slot) -> io::Result<UnixTimestamp> {
+        let params = json!(slot);
+        // At such time when cluster timing varies after genesis, this method will need to be
```

Comment:
I'm a little confused by this comment

Reply:
It's probably not super relevant right now, since we're not actually logging/tracking times from validators. I'll remove.